setup.cfg

```diff
@@ -57,6 +56,8 @@ pyside2 =
     pyside2>=5.0
 pyqt5 =
     pyqt5
+bottleneck =
```

Comment:
Can we update the docs on installation to reflect these changes. In other words, we need to mention bottleneck there, we should mention how to install (both pip and developer etc.) with bottleneck, and also without bottleneck.